lib/db.js

```diff
@@ -26,23 +27,17 @@ connect();
  * Connect to the database.
  */
 function connect() {
-	var connection = config.mongoConnection;
-	MongoClient.connect(config.mongoConnection, {w: 1}, function(error, result) {
-		if (config.mongoConnection != connection) {
-			// connection string changed on the fly; close and ignore
-			log.debug('Discarding open connection to %s', connection);
-			result.close();
-			return;
-		}
+	MongoClient.connect(connectionString, {w: 1}, function(error, result) {
 		if (error) {
 			log.error('Could not connect to MongoDB: %s', error);
 		} else {
 			//close db before assigning a new one
-			log.debug('opening database: ' +  config.mongoConnection);
+			log.debug('opening database: ' +  connectionString);
 			if (db) {
 				db.close(true);
 			}
 			db = result;
+			console.log('db: %s', db);
```

Comment:
I was just debugging an issue with tests, done in commit 9600516cc052617a69b5d550639623149246a49f.